### PR TITLE
Review package and fix some bugs

### PR DESCRIPTION
**Pull request by rodfersou**

### CODE REVIEW

File: {{cookiecutter.package_name}}/src/{{cookiecutter.namespace}}/{{cookiecutter.name}}/content.py [hunk 1]
```diff
@@ -6,4 +6,4 @@
 
 @implementer(IHomepage)
 class Homepage(Container):
-  """"""
+  """Homepage content type"""
```
Comment (rodfersou) [inline {{cookiecutter.package_name}}/src/{{cookiecutter.namespace}}/{{cookiecutter.name}}/content.py]: There are a hidden exception about no docstring in the content type class that makes default view don't work.

File: {{cookiecutter.package_name}}/src/{{cookiecutter.namespace}}/{{cookiecutter.name}}/interfaces.py [hunk 2]
```diff
@@ -4,10 +4,9 @@
 from plone.supermodel import model
 
 
-class I{{cookiecutter.project_slug.capitalize()}}CoreLayer(IDefaultBrowserLayer):
+class IBrowserLayer(IDefaultBrowserLayer):
```
Comment (rodfersou) [inline {{cookiecutter.package_name}}/src/{{cookiecutter.namespace}}/{{cookiecutter.name}}/interfaces.py]: Looks a lot easier like this to me.. please tell me if it is not best practices

File: cookiecutter.json [hunk 3]
```diff
@@ -2,6 +2,6 @@
   "package_name": "package.name",
   "namespace": "{{ cookiecutter.package_name.split('.')[0] }}",
   "name": "{{ cookiecutter.package_name.split('.')[1] }}",
-  "project_slug": "{{ cookiecutter.package_name.lower().replace('.', '') }}",
+  "project_slug": "{{ cookiecutter.name.lower() }}",
```
Comment (rodfersou) [inline cookiecutter.json]: looks a lot better to me.. let me know if I keep like this or change

File: {{cookiecutter.package_name}}/MANIFEST.in [hunk 4]
```diff
@@ -1,4 +1,4 @@
-graft src/{{cookiecutter.package_name}}
+graft src/{{cookiecutter.package_name.replace('.', '/')}}
```
Comment (rodfersou) [inline {{cookiecutter.package_name}}/MANIFEST.in:1]: Bug when release code (don't add configure.zcml)

File: cookiecutter.json (hunk 3 above)
Comment (tisto) [reply cookiecutter.json]: It is unclear to me from the name what this is.

File: {{cookiecutter.package_name}}/src/{{cookiecutter.namespace}}/{{cookiecutter.name}}/browser/configure.zcml [hunk 5]
```diff
@@ -8,12 +8,12 @@
   <include package="z3c.jbot" file="meta.zcml" />
   <browser:jbot
       directory="overrides"
-      layer="{{cookiecutter.package_name}}.interfaces.I{{cookiecutter.project_slug.capitalize()}}CoreLayer"
+      layer="..interfaces.I{{cookiecutter.project_slug.capitalize()}}CoreLayer"
```
Comment (tisto) [inline {{cookiecutter.package_name}}/src/{{cookiecutter.namespace}}/{{cookiecutter.name}}/browser/configure.zcml]: No relative imports anywhere please! That's an anti-pattern.

File: {{cookiecutter.package_name}}/src/{{cookiecutter.namespace}}/{{cookiecutter.name}}/profiles/default/types/Homepage.xml [hunk 6]
```diff
@@ -37,7 +37,6 @@
     <element value="plone.app.contenttypes.behaviors.richtext.IRichText"/>
     <element value="plone.app.content.interfaces.INameFromTitle"/>
     <element value="plone.app.dexterity.behaviors.exclfromnav.IExcludeFromNavigation"/>
-    <element value="plone.app.dexterity.behaviors.id.IShortName"/>
```
Comment (tisto) [inline {{cookiecutter.package_name}}/src/{{cookiecutter.namespace}}/{{cookiecutter.name}}/profiles/default/types/Homepage.xml]: Why did you remove that behavior?

**Review by tisto — CHANGES_REQUESTED** [review]
To be honest I don't see any real bugfixes here. We made the design decisions for a reason and followed Plone core dev best practices.

File: {{cookiecutter.package_name}}/src/{{cookiecutter.namespace}}/{{cookiecutter.name}}/browser/configure.zcml (hunk 5 above)
Comment (rodfersou) [reply {{cookiecutter.package_name}}/src/{{cookiecutter.namespace}}/{{cookiecutter.name}}/browser/configure.zcml]: done

File: cookiecutter.json (hunk 3 above)
Comment (rodfersou) [reply cookiecutter.json]: it is in the line right above, just taking off the module name from the slug, instead of ``collective.faq`` slug would be just ``faq``

File: {{cookiecutter.package_name}}/src/{{cookiecutter.namespace}}/{{cookiecutter.name}}/profiles/default/types/Homepage.xml (hunk 6 above)
Comment (rodfersou) [reply {{cookiecutter.package_name}}/src/{{cookiecutter.namespace}}/{{cookiecutter.name}}/profiles/default/types/Homepage.xml]: added back

Comment (rodfersou) [reply {{cookiecutter.package_name}}/src/{{cookiecutter.namespace}}/{{cookiecutter.name}}/profiles/default/types/Homepage.xml]: I just didn't find it useful..

**Comment by rodfersou** [comment]
@tisto did the changes requested. The bug fixes are:

- Fix Manifest to include the right files https://github.com/collective/collective.faq/issues/3
- Fix default content type view (adding default docstring in the content type)
- Fix robot test comment on how to run tests
- Fix locale configuration, in some places generated ``collective.faq`` other places ``collectivefaq`` so it just don't work the way the code is generated now.

**Comment by rodfersou** [comment]
@tisto Those fixes are all observed on my work with ``collective.faq``.

File: cookiecutter.json (hunk 3 above)
Comment (sneridagh) [reply cookiecutter.json]: I liked the way it was before, because this generates (for a `package.name` package):

```
layer="package.name.interfaces.INameCoreLayer"
```

For example, in Plone we always generate the whole name like:
https://github.com/plone/plone.app.contenttypes/blob/master/plone/app/contenttypes/interfaces.py#L5

```
class IPloneAppContenttypesLayer(Interface):
```

Comment (sneridagh) [reply cookiecutter.json]: I will change it and merge.

**Review by sneridagh — APPROVED** [review]

File: {{cookiecutter.package_name}}/src/{{cookiecutter.namespace}}/{{cookiecutter.name}}/browser/configure.zcml [hunk 7]
```diff
@@ -13,7 +13,7 @@
 
   <!-- Publish static files -->
   <browser:resourceDirectory
-      name="{{cookiecutter.project_slug}}"
+      name="{{cookiecutter.package_name}}"
```
Comment (sneridagh) [inline {{cookiecutter.package_name}}/src/{{cookiecutter.namespace}}/{{cookiecutter.name}}/browser/configure.zcml]: Not a fan of having dots in the resourcedirectories names. I found several issues when using them.

**Comment by rodfersou** [comment]
thanks
